openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -76,12 +76,15 @@ class Meta:
     display_number_with_default = TextField()
     display_org_with_default = TextField()
 
-    # Start/end dates
-    # TODO Remove 'start' & 'end' in removing field in column renaming, DE-1822
     start = DateTimeField(null=True)
     end = DateTimeField(null=True)
+
+    # These are deprecated and unused, but cannot be dropped via simple migration due to the size of the downstream
+    # history table. See DENG-19 for details.
+    # Please use start and end above for these values.
     start_date = DateTimeField(null=True)
     end_date = DateTimeField(null=True)
```

Comment:
Is it possible to do fancy things here, like have the django field names be `_start_date` and `_end_date` so it's very clear that they are not for use, but set the `dbcolumn` as `start_date` and `end_date` maybe?

This comment is likely good enough, just brainstorming.

Reply:
If memory serves, that might still generate a django migration (though I don't think it would change the database itself when run). But maybe not worth the risk.